Makefile

```diff
@@ -19,7 +19,7 @@ compile:
 
 test:
 	@echo "Testing..."
-	$(EASK) test ert ./test/*.el
+	$(EASK) test
```

Comment:
The command should change to `eask run script test`.

Reply:
Thanks! I made the update using your suggestion.